src/helpers/discord_utils.ts

```diff
@@ -439,6 +439,21 @@ export async function sendMessage(
     authorID?: string,
     interaction?: Eris.ComponentInteraction | Eris.CommandInteraction,
 ): Promise<Eris.Message | null> {
+    if (messageContent.messageReference) {
+        const message: Message<TextChannel> | undefined = await (
+            State.client.getChannel(textChannelID!) as
+                | Eris.TextChannel
+                | undefined
+        )?.getMessage(messageContent.messageReference.messageID);
```

Comment:
Will this lookup slow down non-test send message calls

Reply:
These are cached